dev_tools/Dockerfile

```diff
@@ -11,14 +21,24 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8 
 
 # Make python3 default
-RUN rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python
+RUN rm -f /usr/bin/python \
+     && ln -s /usr/bin/python3 /usr/bin/python
 
 # Get a current copy of Cirq from Github.
 RUN git clone https://github.com/quantumlib/Cirq
 
+COPY requirements.txt .
+COPY Cirq/cirq/contrib/contrib-requirements.txt .
```

Comment:
But these two should change and you could just use `./cirq/contrib/contrib-requirements` and `./dev_tools/conf/pip-list-dev-tools.txt`.